test/TesterInternal/DevTestClientConfiguration.xml

```diff
@@ -1,7 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <ClientConfiguration xmlns="urn:orleans">
   <SystemStore SystemStoreType="SqlServer"
-               DataConnectionString="connection string"/>
+               DataConnectionString="connection string"
+               DeploymentId="deployment id"/>
```

Comment:
It is used in config test, the test was just testing loading config from a file works or not. And the test is failing in main solution too. So I just fixed it. 

Let's talk about it tomorrow.